webapp/client/cypress/integration/filing.spec.js

```diff
@@ -0,0 +1,64 @@
+describe("Filing", () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  context("success", () => {
+    beforeEach(() => {
+      cy.fixture("est_data").then((est_data) => {
+        cy.request("POST", "/testing/set_session_data/form_data", est_data);
+      });
+      cy.visit("/lotse/step/filing");
+    });
+
+    it("downloading pdf is possible", () => {
+      cy.window()
+        .document()
+        .then(function (doc) {
+          doc.addEventListener("click", () => {
+            setTimeout(function () {
+              doc.location.reload();
+            }, 1000);
+          });
+          // Downloads folder is automatically cleared
+          cy.get("a").contains("Übersicht speichern").click();
+          cy.readFile("cypress/downloads/AngabenSteuererklaerung.pdf").should(
+            "exist"
+          );
+        });
```

Comment:
Clever idea 👍 
We could do this also similar to the way we do it in the acceptance tests:

```javascript
cy.get("a").contains("Übersicht speichern");
cy.request('/download_pdf/print.pdf').its('body').should('not.be.empty')
```
or:
```javascript
cy.get("a").contains("Übersicht speichern").should('have.attr', 'href').and("include", "download_pdf/print.pdf");
     .then((href) => {
         cy.request(href).its('body').should('not.be.empty')
     })
```

We then do not need to set the timeout

Reply:
Second option is good!